src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/HtmlHelperFormExtensions.cs

```diff
@@ -308,6 +394,7 @@ public static MvcForm BeginRouteForm(this IHtmlHelper htmlHelper, object routeVa
                 routeName: null,
                 routeValues: routeValues,
                 method: FormMethod.Post,
+                antiforgery: false,
```

Comment:
Good catch. We changed the polarity of this during design disucssions. I missed this one.